src/sagemaker_inference/model_server.py

```diff
@@ -150,6 +150,7 @@ def _generate_mms_config_properties():
         "default_workers_per_model": env.model_server_workers,
         "inference_address": "http://0.0.0.0:{}".format(env.inference_http_port),
         "management_address": "http://0.0.0.0:{}".format(env.management_http_port),
+        "vmargs": "-XX:-UseContainerSupport",
```

Comment:
Wondering if we should make this configurable via SM `env`? Not sure if it would require additional changes anywhere else +@dhanainme 
`"vmargs": env.vmargs if env.vmargs else "-XX:-UseContainerSupport"`

Reply:
+1. TS_VM_ARGS could be the env variable where we can pick up from

Reply:
This change needs to happen here in this file

https://github.com/aws/sagemaker-inference-toolkit/blob/cb9e793a79ef4dbc165b0cc48d3c6202916cea33/src/sagemaker_inference/environment.py#L66

Reply:
We have tried the fix in this PR that should sovle #82 but we see no difference in the number of CPU's logged in cloudwatch. So not sure if more changes are involved, but this fix as seperate change seems not solving the issue. The container we use (pytorch 1.7.1, torch-serve 0.4.0), uses JDK 11 which should have the property `-XX:-UseContainerSupport` enabled by default.

Reply:
I think it won't work for PyTorch >= 1.6 containers since `torchserve` model server is used

Reply:
The reason why this doesn't fix PT >=1.6 is because the [pytorch inference toolkit](https://github.com/aws/sagemaker-pytorch-inference-toolkit/blob/6610a410c0cf40bcf15267abe722d20d50e77bcf/src/sagemaker_pytorch_serving_container/torchserve.py#L144-L172) needs similar fix.